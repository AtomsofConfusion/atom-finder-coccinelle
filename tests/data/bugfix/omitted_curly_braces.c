#include <stdio.h>

int main() {
    int a = 10, b = 20;
    int i = 0, j = 10;

    if (a < b)
        a++;
    else
        a--, 
    b--;

    while (a) a--, b--;


    // this is fine
    if (a < b) {
        a--, 
        b--;
    }
    
    // this is not
    if (a < b)
        a--, 
    b--;
    
    while (i < j)
        a ++;

    while (i < j)
        a--, 
    b--;

    for (i = 0; i < j; i ++ ){
        a--, 
        b--;
    }

    for (i = 0; i < j; i ++ )
        a--;
    b--;

    for (i = 0; i < j; i ++ )
        a--,
    b--;

    return 0;

}


