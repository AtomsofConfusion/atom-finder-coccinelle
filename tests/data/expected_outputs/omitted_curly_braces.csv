omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,8,8,"printf ( ""'if' without braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,14,8,"printf ( ""'if' without braces before an else"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,24,8,"printf ( ""'if' wuthout braces with an else after it"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,83,8,"printf ( ""else if with curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,85,8,"printf ( ""else if with curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,32,8,"printf ( ""else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,52,8,"printf ( ""else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,70,8,"printf ( ""else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,72,8,"printf ( ""else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,11,8,"printf ( ""if without braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,30,8,"printf ( ""if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,50,8,"printf ( ""if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,68,8,"printf ( ""if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,96,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,98,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,112,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,114,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,128,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,130,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,145,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,147,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,163,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,165,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,179,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,181,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,195,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,197,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,211,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,213,12,"printf ( ""nested else if without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,193,12,"printf ( ""nested if with curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,192,8,"if ( x > 4 ) printf ( ""nested if with curly braces"" ) ; else if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }     "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,160,8,"if ( x > 19 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 18 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 17 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }     "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,176,8,"if ( x > 9 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 8 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }     "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,21,8,"printf ( ""'else' without braces after if with braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,16,8,"printf ( ""'else' without braces after if without braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,54,8,"printf ( ""else without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,74,8,"printf ( ""else without curly braces"" ) ;"
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,31,9,"if ( x < y ) printf ( ""else if without curly braces"" ) ; "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,36,11,"if ( x < y ) { printf ( ""else if with curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,71,9,"if ( x > 0 ) printf ( ""else if without curly braces"" ) ; else printf ( ""else without curly braces"" ) ; "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,51,9,"if ( x == y ) printf ( ""else if without curly braces"" ) ; else printf ( ""else without curly braces"" ) ; "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,84,9,"if ( x > 6 ) printf ( ""else if with curly braces"" ) ; else { printf ( ""else with curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,61,11,"if ( x > 0 ) { printf ( ""else if with curly braces"" ) ; } else { printf ( ""else with curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,133,15,"if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,200,15,"if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,101,15,"if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,168,15,"if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,117,15,"if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,184,15,"if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,150,15,"if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,216,15,"if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,43,11,"if ( x == y ) { printf ( ""else if with curly braces"" ) ; } else { printf ( ""else with curly braces"" ) ; } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,69,9,"if ( x > 5 ) printf ( ""else if without curly braces"" ) ; else if ( x > 0 ) printf ( ""else if without curly braces"" ) ; else printf ( ""else without curly braces"" ) ;  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,82,11,"if ( x > 7 ) printf ( ""else if with curly braces"" ) ; else if ( x > 6 ) printf ( ""else if with curly braces"" ) ; else { printf ( ""else with curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,131,13,"if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,198,13,"if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,99,13,"if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,166,13,"if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,59,11,"if ( x > 5 ) { printf ( ""else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""else if with curly braces"" ) ; } else { printf ( ""else with curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,115,13,"if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,182,13,"if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,148,13,"if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,214,13,"if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,80,11,"if ( x > 8 ) { printf ( ""else if with curly braces"" ) ; } else if ( x > 7 ) printf ( ""else if with curly braces"" ) ; else if ( x > 6 ) printf ( ""else if with curly braces"" ) ; else { printf ( ""else with curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,97,13,"if ( x > 17 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,164,13,"if ( x > 17 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,129,13,"if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,196,13,"if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,113,13,"if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,180,13,"if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,146,13,"if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,212,13,"if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }   "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,95,15,"if ( x > 18 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 17 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,162,15,"if ( x > 18 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 17 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 16 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 15 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,127,15,"if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,194,13,"if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,111,15,"if ( x > 8 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,178,15,"if ( x > 8 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,78,11,"if ( x > 9 ) { printf ( ""else if with curly braces"" ) ; } else if ( x > 8 ) { printf ( ""else if with curly braces"" ) ; } else if ( x > 7 ) printf ( ""else if with curly braces"" ) ; else if ( x > 6 ) printf ( ""else if with curly braces"" ) ; else { printf ( ""else with curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,144,15,"if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,210,15,"if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }    "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,208,8,"if ( x > - 1 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }     "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,190,9,"if ( x > 5 ) if ( x > 4 ) printf ( ""nested if with curly braces"" ) ; else if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      else if ( x > - 1 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,123,11,"if ( x > 5 ) { if ( x > 4 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      } else { if ( x > - 1 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      } "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,174,9,"if ( x > 10 ) if ( x > 9 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 8 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      else if ( x > 5 ) if ( x > 4 ) printf ( ""nested if with curly braces"" ) ; else if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      else if ( x > - 1 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }       "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,107,11,"if ( x > 10 ) { if ( x > 9 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 8 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 7 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 6 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      } else if ( x > 5 ) { if ( x > 4 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > 1 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > 0 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      } else { if ( x > - 1 ) { printf ( ""nested if with curly braces"" ) ; } else if ( x > - 2 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 3 ) printf ( ""nested else if without curly braces"" ) ; else if ( x > - 4 ) { printf ( ""nested else if with curly braces"" ) ; } else if ( x > - 5 ) { printf ( ""nested else if with curly braces"" ) ; } else { printf ( ""nested else without curly braces"" ) ; }      }  "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,223,4,"while ( x < 5 ) printf ( ""'while' without braces, x: %d"" , x ++ ) ; "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,237,4,"do printf ( ""do while iteration: %d"" , x ++ ) ; while ( x < 20 ) ; "
omitted-curly-braces,/home/ubuntu/atom-finder-coccinelle/tests/data/inputs/omitted_curly_braces.c,230,4,"for (  int i = 0 ; i < 5 ; i ++ ) printf ( ""'for' iteration: %d"" , i ) ; "

